barretenberg/cpp/src/barretenberg/dsl/acir_format/bigint_constraint.hpp

```diff
@@ -126,11 +126,11 @@ template <typename Builder> class DSLBigInts {
     void set_value(uint256_t value, const std::array<uint32_t, 5> limbs_idx)
     {
         uint256_t limb_modulus = uint256_t(1) << big_bn254_fq::NUM_LIMB_BITS;
-        builder->variables[limbs_idx[4]] = value;
```

Comment:
The usage here is incorrect. If this witness is involved in a nontrivial copy constraint (such that real_variables_idx[idx] != idx) then this line will not update the correct witness value.